server/.env

```diff
@@ -1 +1,2 @@
 TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam"
+DB=<database URL>
```

Comment:
You can remove this change as Lin's PR will remove the .env file anyway. Just message the Slack what we all should add to the env file

Reply:
ok sounds good